assets/js/menu.js

```diff
@@ -44,73 +45,74 @@ class Menu {
   }
 
   static reorderMenu(menu) {
-    $(menu).children().sort(function (a, b) {
-      return a.textContent.toLowerCase().localeCompare(b.textContent.toLowerCase());
-    }).appendTo(menu);
+    Array.from(menu.children)
+      .sort((a, b) => a.textContent.toLowerCase().localeCompare(b.textContent.toLowerCase()))
+      .forEach((child) => menu.appendChild(child));
   }
 
   static addCycleWarning(element, isSameCycle) {
-    const hasCycleWarning = $(`${element} .same-cycle-warning-menu`).length > 0;
-    const category = $(element);
+    const category = document.querySelector(element);
+    const parentParent = category.parentElement.parentElement;
+    const hasCycleWarning = parentParent.querySelector('.same-cycle-warning-menu') !== null;
     if (isSameCycle && !hasCycleWarning) {
-      category.parent().parent().attr('data-help', 'item_category_same_cycle');
-      category.append(`<img class="same-cycle-warning-menu" src="./assets/images/same-cycle-alert.png">`);
+      parentParent.setAttribute('data-help', 'item_category_same_cycle');
+      const img = document.createElement('img');
+      img.classList.add('same-cycle-warning-menu');
+      img.src = './assets/images/same-cycle-alert.png';
+      category.appendChild(img);
     } else if (!isSameCycle && hasCycleWarning) {
-      category.parent().parent().attr('data-help', 'item_category');
-      category.children('.same-cycle-warning-menu').remove();
+      parentParent.setAttribute('data-help', 'item_category');
+      const cycleWarning = category.querySelector('.same-cycle-warning-menu');
+      cycleWarning.parentNode.removeChild(cycleWarning);
     }
   }

```

Comment:
When cycles are not updated (in cycles.json), default behavior was to show cycles warning icon on the right from the category name and set each cycle to 1. It doesn't work now.